test/support/factories/accounts_factory.ex

```diff
@@ -13,6 +13,7 @@ defmodule Bokken.Factories.AccountFactory do
         %User{
           email: sequence(:email, &"email-#{&1}@mail.com"),
           password_hash: Argon2.hash_pwd_salt("password1234!"),
+          password: "password1234!",
```

Comment:
Why is this addition needed?

Reply:
because if you do `params_for(:user)` it won't add a `:password`, only `:password_hash`, which the schema doesn't recognize

Reply:
A really good way to speed this up would be:

```elixir
@password "password1234!"
@hashed_password Argon2.hash_pwd_salt(@password)
```

That way, all users from the factory would share the same hashed password (which is a really slow calculation) making every single test much faster 💪